trollsched/schedule.py

```diff
@@ -96,7 +95,7 @@ def single_station(self, sched, start_time, tle_file):
 
         allpasses = self.get_next_passes(opts, sched, start_time, tle_file)
 
-        area_boundary = AreaDefBoundary(self.area, frequency=500)
+        area_boundary = self.area.boundary(8)
```

Comment:
Do we have a test for this?

Reply:
well, I changed this because of a deprecation warning from pyresample, and the test I wrote (that check that generating a schedule works) passes, which wouldn't if this call was wrong...